SeaPublicWebsite/Views/Feedback/FeedbackForm.cshtml

```diff
@@ -0,0 +1,86 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model SeaPublicWebsite.Models.Feedback.FeedbackFormViewModel
+@{
+    ViewBag.Title = "Help us improve GOV.UK";
+}
+
+@section BeforeMain {
+    @await Html.GovUkBackLink(new BackLinkViewModel
+    {
+        Text = "Back",
+        OverrideWithJavascript = true
```

Comment:
This returns the user to the page he came from.
If we want to do it without JS It is too much work to be worth it.